sched/scheduler/scheduler_test.go

```diff
@@ -2,6 +2,8 @@ package scheduler
 
 import (
 	"fmt"
+	"testing"
```

Comment:
Ohhhh! goimports!

Yes, gofmt preserves blocks, so if you leave a blank line, it will sort them within that block.